wapitiCore/attack/mod_https_redirect.py

```diff
@@ -27,6 +27,19 @@ def test_port(address: str, dest_port: int, timeout: float = None) -> bool:
         return False
 
 
+async def check_http_redirection(self, request: Request):
+    try:
+        response: Response = await self.crawler.async_send(request, follow_redirects=False)
+    except RequestError:
+        self.network_errors += 1
+        return
+    if response.status in (301, 302):
```

Comment:
You can simplify this by using attributes `is_redirect` and `redirection_url` on the response object